.github/workflows/ci.yml

```diff
@@ -85,3 +85,29 @@ jobs:
         with:
           severity: error
           ignore_paths: third-party
+  dokken-recipes:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os:
+          - alinux2
+          - centos7
+          - ubuntu18
+          - ubuntu20
```

Comment:
Should we also add `redhat8`?

Reply:
yes, but probably is not so easy, so I'll do it later (if I have time) for now at least we're sure we're not breaking existing OSes :)

Reply:
I surely miss something: why shouldn't it be easy, given the set of tests we run? What is the hard part?

Reply:
The hard part is that the redhat8 image was not available on dokken (locally we're building it). Anyway I found the right base image to use, see patch: https://github.com/aws/aws-parallelcluster-cookbook/pull/1647